doc/configuration.rst

```diff
@@ -1741,6 +1742,49 @@ flag is passed to ``sphinx-build``. This can be enabled by setting::
     }
 
 
+.. _recommend_examples:
+
+Enabling the example recommender system
+=======================================
+
+Sphinx-Gallery can be configured to generate content-based recommendations for
+the example gallery. A list of related examples is automatically generated by
+computing the closest examples in the TF-IDF space of their text contents. Then,
```

Comment:
Maybe link "TF-IDF" to the wikipedia article: https://en.wikipedia.org/wiki/Tf%E2%80%93idf